data/hai/culture hai.txt

```diff
@@ -0,0 +1,208 @@
+			choice
+				`	"How strange."`
+				`	"Sounds fun."`
+			label done
+			`	You thank the security guard for his advice and go about what you were doing as the baffling conflict continues around you.`
+				decline
+
+mission "ZZCU Wrestling Star"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`A famous Hai wrestler is doing a signing. (TBD)`
+				decline
+
+mission "ZZCU Fencing Star"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`A famous Hai fencer is doing a signing. (TBD)`
+				decline
+				
+mission "ZZCU ??? Star"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`A famous Hai sportsperson of an obscure alien sport is doing a signing. (TBD)`
+				decline
+
+mission "ZZCU Gambling Judgment"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		random == 1
+	on offer
+		conversation
+			`Upon entering a diner at the spaceport you have not been seated long before you notice there is a group of other human ship captains sequestered in a corner playing, near as you can tell, a game of poker. You note that the group seems to be getting a variety of dark looks from various Hai also dining at the establishment.`
+			`	After a short while the barkeep, a middling and slightly overweight Hai, comes over to take your order.`
+			choice
+				`	"Just some food thanks." (Place an order.)`
+					goto end
+				`	"Hey, I notice there are some dark looks around here. Are human captains not welcome?"`
+			`	The barkeep glances over at the table and shakes his head, "Oh no, you're perfectly welcome. That group is just breaking the maturity tenets, some Hai get uncomfortable with adults blatantly acting like children in public."`
+			choice
+				`	"OK..." (Place an order.)`
+					goto end
+				`	"The maturity tenets?"`
+			`	"Oh right, I suppose you wouldn't know, there are a number of tenets that Hai are taught growing up, lessons to be internalized to be a responsible and functioning member of society. The second is 'To be honest in dealings.' and the third is 'To gamble only on skill, not chance.' so their game breaks two of them."`
+			choice
+				`	"Some would say Poker is more about skill than chance."`
+				`	"Surely if it's just a game though, there's no harm in it?"`
+			`	"Oh no, I'm afraid the first tenet ensures that it's really not favorable either way. The first is 'To have clear separation of fantasy and reality.' so a game cannot have real consequences or it is reality, not fantasy. So to gamble money on the game is to bring real consequences to a fantasy exercise, this is taboo for an adult.`
```

Comment:
I'm just going to say - People don't necessarily speak and present information logically.
But if you think there are a collection of grammar edits needed please use the function to suggest en edit.

Reply:
> I'm just going to say - People don't necessarily speak and present information logically.

This is a terrible approach to writing anything. In movies people don't interrupt others, forget what they're saying, etc. unless it serves the plot. If someone coughs in a story they are either dying or they have some disease which is important to the plot. This is not a flaw, it is by design. You shouldn't excuse wonky writing with "realism", because if you actually wanted to write exactly how people speak it would suck to read.